src/containers/Downloads/index.js

```diff
@@ -95,12 +112,12 @@ class Download extends Component {
   };
 
   renderExperimentsView = () => {
-    const { dataSet } = this.props;
+    const { dataSet, experiments } = this.props;
 
     if (!Object.keys(dataSet).length)
       return <p>No samples added to download dataset.</p>;
     return Object.keys(dataSet).map((id, i) => {
```

Comment:
So I was initially using the separate `experiments` and `samples` objects for rendering this view, but I realized it got super messy with adding and deleting of experiments and samples because I had to update three separate objects. I found this to be a better way of having one source of truth and filtering the other objects for experiments and samples in the `dataSet`.